code/game/objects/items/weapons/manuals.dm

```diff
@@ -809,6 +844,44 @@
 		</html>
 
 		"}
+	var/selected_law = null
+	var/last_hit = 0
+
+/obj/item/book/manual/security_space_law/attack(mob/living/M, mob/living/user, def_zone)
+	. = ..()
+	if(ishuman(M))
+		if(last_hit < world.time - 30) // Every 3 seconds you can do this
+			last_hit = world.time
+			var/law = selected_law ? selected_law : pick(space_laws)
+			show_law(M, law)
+		else
+			if(prob(33))
+				playsound(user, 'sound/weapons/Genhit.ogg', 50, 1)
+				user.Stun(3)
+				user.Weaken(3)
+				user.apply_effect(STUTTER, 3)
+				playsound(loc, 'sound/weapons/Egloves.ogg', 50, 1, -1)
+				var/law = "106, Abuse of Equipment" // Fitting don't you think?
+				show_law(user, law)
+				if(prob(20))
+					user.adjustBrainLoss(5)
+					user.say("LAW!!!", null, "screams")
+
+/obj/item/book/manual/security_space_law/proc/show_law(mob/living/carbon/human/H, law)
+	H.show_message("<span class='danger'>You hear a stern voice in your head!</span>")
+	H.show_message("<span class='danger'>[law]. [space_laws[law]]</span>")
+
+/obj/item/book/manual/security_space_law/AltClick(mob/user)
+	if(ishuman(user) && ishuman(loc))

```

Comment:
An `Adjacent()` check would be nicer I think than `ishuman(loc)`

Reply:
Changing this to Adjacent will also make you proc it when the book is on the ground. Which can be annoying if you want to view the items on the ground.

Reply:
That's why I said it so that you can interact with the book while not actually carrying it. But if that's by design then sure.